lib/rubocop/ast/node/regexp_node.rb

```diff
@@ -32,6 +32,11 @@ def content
         children.select(&:str_type?).map(&:str_content).join
       end
 
+      # @return [Bool] if char is one of the delimiters
+      def delimiter?(char)
```

Comment:
I guess you can also add a `delimiters` method that returns the two of them and implement this one in terms of it.

Reply:
Good idea - I took a quick look for related predicates used by RuboCop, so added those too, let me know what you think